SeaPkg/Library/BasePeCoffLibNegative/BasePeCoffLibNegative.c

```diff
@@ -884,7 +884,7 @@ PeCoffImageDiffValidation (
         NextImageValidationEntryHdr = (IMAGE_VALIDATION_ENTRY_HEADER *)((IMAGE_VALIDATION_MEM_ATTR *)ImageValidationEntryHdr + 1);
         break;
       case IMAGE_VALIDATION_ENTRY_TYPE_SELF_REF:
-        Status                      = PeCoffImageValidationSelfRef (TargetImage, ImageValidationEntryHdr, OriginalImageBaseAddress);
+        Status                      = PeCoffImageValidationSelfRef (OriginalImageBaseAddress, ImageValidationEntryHdr, OriginalImageBaseAddress);

```

Comment:
Does this API need to be updated to only accept a single image base address argument?

Reply:
I would be OK with this

Reply:
Probably a good idea.  I'll make that update.

Reply:
You will need to update the header file as well.